Resources/views/Collector/db.html.twig

```diff
@@ -1,23 +1,45 @@
 {% extends app.request.isXmlHttpRequest ? 'WebProfilerBundle:Profiler:ajax_layout.html.twig' : 'WebProfilerBundle:Profiler:layout.html.twig' %}
 
+{% set profiler_markup_version = profiler_markup_version|default(1) %}
```

Comment:
and actually, it is needed only inside the toolbar block for now, as the profiler itself does not yet have a new markup and so we don't need this variable yet there